demisto_sdk/commands/common/schemas/script.yml

```diff
@@ -164,7 +161,5 @@ schema;important_schema:
     description:
       type: str
       required: True
-    description_x2:
```

Comment:
We did think at the start that we will need to change the description too (if it had incidents instead of alerts), but then decided to stay with the id and name.